[2023-03-29 17:06:21,660] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE.extract_data_task manual__2023-03-29T17:06:07.245402+00:00 [queued]>
[2023-03-29 17:06:21,710] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE.extract_data_task manual__2023-03-29T17:06:07.245402+00:00 [queued]>
[2023-03-29 17:06:21,713] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 17:06:21,721] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 17:06:21,723] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 17:06:21,832] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): extract_data_task> on 2023-03-29 17:06:07.245402+00:00
[2023-03-29 17:06:21,847] {standard_task_runner.py:52} INFO - Started process 364 to run task
[2023-03-29 17:06:21,859] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE', 'extract_data_task', 'manual__2023-03-29T17:06:07.245402+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpmtxkzaac', '--error-file', '/tmp/tmph05kuv76']
[2023-03-29 17:06:21,873] {standard_task_runner.py:77} INFO - Job 158: Subtask extract_data_task
[2023-03-29 17:06:22,064] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE.extract_data_task manual__2023-03-29T17:06:07.245402+00:00 [running]> on host eebc01130c53
[2023-03-29 17:06:22,265] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE
AIRFLOW_CTX_TASK_ID=extract_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T17:06:07.245402+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T17:06:07.245402+00:00
[2023-03-29 17:06:29,635] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas_datareader/tiingo.py:234: FutureWarning: In a future version of pandas all arguments of concat except for the argument 'objs' will be keyword-only
  return pd.concat(dfs, self._concat_axis)

[2023-03-29 17:06:29,687] {logging_mixin.py:109} INFO - ingestion from api completed
[2023-03-29 17:06:29,812] {logging_mixin.py:109} INFO - file has been saved at :sp_500_data.csv
[2023-03-29 17:06:29,814] {logging_mixin.py:109} INFO - sp_500_data.csv
[2023-03-29 17:06:29,816] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 17:06:29,859] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE, task_id=extract_data_task, execution_date=20230329T170607, start_date=20230329T170621, end_date=20230329T170629
[2023-03-29 17:06:29,957] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 17:06:30,099] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
